diff -aurN keepassx-0.4.3.orig/src/lib/random.cpp keepassx-0.4.3/src/lib/random.cpp
--- keepassx-0.4.3.orig/src/lib/random.cpp	2013-01-21 11:11:00.417557788 +0100
+++ keepassx-0.4.3/src/lib/random.cpp	2013-01-21 11:11:32.562791462 +0100
@@ -18,9 +18,9 @@
  *   59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.             *
  ***************************************************************************/
 
+#include <unistd.h>
 #include "random.h"
 
-
 #if defined(Q_WS_X11) || defined(Q_WS_MAC)
 	#include <QFile>
 #elif defined(Q_WS_WIN)
